src/components/Plugin/ContributionRewardExtRewarders/Competition/Details.tsx

```diff
@@ -276,7 +275,7 @@ class CompetitionDetails extends React.Component<IProps, IStateProps> {
     this.disqusConfig.identifier = `competition-${proposalState.id}`;
 
     return <React.Fragment>
-      <BreadcrumbsItem weight={1} to={`/dao/${daoState.address}/plugin/${proposalState.plugin.id}/crx`}>{pluginName(proposalState.plugin.entity.coreState, proposalState.plugin.entity.coreState.address)}</BreadcrumbsItem>
+      <BreadcrumbsItem weight={1} to={`/dao/${daoState.address}/plugin/${proposalState.plugin.id}/crx`}>Competition</BreadcrumbsItem>
```

Comment:
does pluginName function has an issue?

Reply:
good catch!  Yes there was something wrong with `pluginName`.

Note that having fixed that, it reveals the problem described in [this ticket](https://github.com/daostack/alchemy/issues/1985), which is a separate problem.